ckanext/qdes_schema/fanstatic/accordion.js

```diff
@@ -0,0 +1,12 @@
+jQuery(document).ready(function () {
+      
+  document.addEventListener('keyup', function (e) {
```

Comment:
I think we can use a better selector on the accordion elements for keyup events instead of the whole document.
Will provide some examples tomorrow